deploy/crds/metrics-aggregation.crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Your CRD should be added to the `templates/crds/` dir.

Reply:
Ack, fixing.